analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -490,7 +490,7 @@ class Pip(
         // "License :: OSI Approved :: GNU Library or Lesser General Public License (LGPL)"
         pkgInfo["classifiers"]?.mapNotNullTo(declaredLicenses) {
             val classifier = it.textValue().split(" :: ")
-            classifier.takeIf { it.first() == "License" }?.last()
+            classifier.takeIf { it.first() == "License" }?.last()?.takeIf { it != "OSI Approved" }
```

Comment:
Nit: Maybe `takeUnless { it == "OSI Approved" }` to make more clear that the expectation is to usually take it?

Reply:
Done.